src/styles/App.scss

```diff
@@ -26,3 +26,7 @@ $navbar-height: 50px;
 .poi-form-container {
 
 }
+
```

Comment:
Remove this, this shouldn't have made it into your commit. We were just playing with it but it doesn't actually work. Also, you should set your editor's indentation to 2 spaces